docs/arm.yaml.sample

```diff
@@ -212,8 +213,10 @@ HB_PRESET_BD: "HQ 1080p30 Surround"
 # Extension of the final video file
 DEST_EXT: mkv
 
-# Handbrake binary to call
+# Handbrake binary to call.  These should be the same unless you are doing remote
+# processing.
 HANDBRAKE_CLI: HandBrakeCLI
+HANDBRAKE_LOCAL: HandBrakeCLI
```

Comment:
Adding `HANDBRAKE_LOCAL` to the arm.yaml means that it will need to be added to the comments.json file with comments, or some of the armui might break.

Adding this
```
"HANDBRAKE_LOCAL": "# Handbrake binary to call.  These should be the same unless you are doing remote processing.",
```
to the comments file should fix it.